.circleci/config.yml

```diff
@@ -248,25 +250,10 @@ jobs:
           command: |
             python -m scripts.run_e2e_tests --skip-install --skip-build --suite="profileFeatures" --prod_env
 
-  e2e_accessibility_and_embedding:
-    <<: *job_defaults
-    steps:
-      - checkout
-      - attach_workspace:
-          at: /home/circleci/
-      - run:
-          name: Run e2e accessibility test
-          command: |
-            python -m scripts.run_e2e_tests --skip-install --skip-build --suite="accessibility" --prod_env
-      - run:
-          name: Run e2e embedding test
-          command: |
-            python -m scripts.run_e2e_tests --skip-install --skip-build --suite="embedding" --prod_env
-
   # Non-prod e2e Tests
   # The following tests must be run in a non-production environment because they are uploading files.
   # All other e2e tests should be run in prod mode.
-  e2e_file_upload_features:
+  e2e_non_prod_tests:
```

Comment:
Similarly we've been seeing flakes in the classroom page tests, so we might want those separate